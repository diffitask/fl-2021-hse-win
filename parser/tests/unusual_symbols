Alphabet: 8 -- ("--", ":", " ", "k", "^", "@", "\"", "\\") 
Q: 2
start: 1
runoff: 2
T: 1 -- 1
edges: (1; 2; "--", ":", " ", "k", "^", "@", "\"", "\\"), (2; 2; "--", ":", " ", "k", "^", "@", "\"", "\\")