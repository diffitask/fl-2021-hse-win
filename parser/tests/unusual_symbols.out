Alphabet -- 8 symbol(s):
             ( , --, :, @, \", \\, ^, k)
Start state:
             1
Terminals -- 1 state(s):
             1
All states:
           (number: 1, type: start)
           (number: 2, type: runoff)
Edges:
            1 --> 2, symbols: ( , --, :, @, \", \\, ^, k)
            2 --> 2, symbols: ( , --, :, @, \", \\, ^, k)
